etrago/tools/plot.py

```diff
@@ -3329,3 +3335,184 @@ def plot_heat_summary(self, t_resolution="20H", stacked=True, save_path=False):
 
     if save_path:
         plt.savefig(save_path, dpi=300)
+
+
+def shifted_energy(self, carrier, buses):
```

Comment:
Since "self" was used as the first parameter, I tried to use the function typing etrago.shifted_energy(...), but it did not work. The function works well when it is imported directly from plot.py.